book-1-orientation/chapters/ADONET_INTRO.md

```diff
@@ -9,624 +9,305 @@ This will be an interactive lesson. Your instructor will lead you through follow
-            repository.UpdateEmployee(dbJane.Id, dbJane);
-
-            employees = repository.GetAllEmployeesWithDepartment();
-            PrintEmployeeReport("All Employees after updating Jane", employees);
-
-            Pause();
-
-
-            repository.DeleteEmployee(dbJane.Id);
-            employees = repository.GetAllEmployeesWithDepartment();
-
-            PrintEmployeeReport("All Employees after updating Jane", employees);
-
-            Pause();
-
-        }
-
-
-
-        /// <summary>
-        ///  Prints a simple report with the given title and department information.
-        /// </summary>
-        /// <remarks>
-        ///  Each line of the report should include the Department's ID and Name
-        /// </remarks>
-        /// <param name="title">Title for the report</param>
-        /// <param name="departments">Department data for the report</param>
-        public static void PrintDepartmentReport(string title, List<Department> departments)
-        {
-            /*
-             * TODO: Complete this method
-             *  For example a report entitled, "All Departments" should look like this:
-
-                All Departments
-                1: Marketing
-                2: Engineering
-                3: Design
-             */
-        }
-
-        /// <summary>
-        ///  Prints a simple report with the given title and employee information.
-        /// </summary>
-        /// <remarks>
-        ///  Each line of the report should include the
-        ///   Employee's ID, First Name, Last Name,
-        ///   and department name IF AND ONLY IF the department is not null.
-        /// </remarks>
-        /// <param name="title">Title for the report</param>
-        /// <param name="employees">Employee data for the report</param>
-        public static void PrintEmployeeReport(string title, List<Employee> employees)
-        {
-            /*
-             * TODO: Complete this method
-             *  For example a report entitled, "All Employees", should look like this:
-
-                All Employees
-                1: Margorie Klingerman
-                2: Sebastian Lefebvre
-                3: Jamal Ross
-
-             *  A report entitled, "All Employees with Departments", should look like this:
-
-                All Employees with Departments
-                1: Margorie Klingerman. Dept: Marketing
-                2: Sebastian Lefebvre. Dept: Engineering
-                3: Jamal Ross. Dept: Design
-
-             */
-        }
-
-
-        /// <summary>
-        ///  Custom function that pauses execution of the console app until the user presses a key
-        /// </summary>
-        public static void Pause()
-        {
-            Console.WriteLine();
-            Console.Write("Press any key to continue...");
-            Console.ReadLine();
-            Console.WriteLine();
-            Console.WriteLine();
-        }
-    }
-}
-```
+   ```sh
+   dotnet add package System.Data.SqlClient
```

Comment:
We should probably switch to `Microsoft.Data.SqlClient`

https://devblogs.microsoft.com/dotnet/introducing-the-new-microsoftdatasqlclient/

Reply:
I don't think dapper or efcore are using the new sqlclient stuff yet, so might still be early.  /shrug

Reply:
IIRC I’ve seen the VS “lightbulb” offer to add a reference Microsoft.Data.Sqlclient.